internal/nodes/dequeue_ingestion.go

```diff
@@ -382,13 +378,28 @@ func handlePod(key, namespace, podName string, fullsync bool) {
 		}
 		if len(lbSvcs) != 0 {
 			objects.SharedPodToLBSvcLister().Save(podKey, lbSvcs)
+			handleLBSvcUpdateForPod(key, lbSvcs, fullsync)
 		}
-		for _, lbSvc := range lbSvcs {
+		utils.AviLog.Infof("key: %s, msg: NPL Services retrieved: %s", key, services)
+	}
+}
+
+func handleLBSvcUpdateForPod(key string, lbSvcs []string, fullsync bool) {
+	for _, lbSvc := range lbSvcs {
+		lbSvcSplit := strings.Split(lbSvc, "/")
+		lbSvcNS, lbSvcName := lbSvcSplit[0], lbSvcSplit[1]
+		sharedVipLBSvcKey := lib.SharedVipServiceKey + "/" + lbSvc
+		sharedVipKeys, keysFound := ServiceChanges(lbSvcName, lbSvcNS, sharedVipLBSvcKey)
```

Comment:
@DixitAakash: We can ignore `keysFound` as function always true.  Also we can take out ` utils.CheckIfNamespaceAccepted(lbSvcNS) ` as all lbsvc are from accepted namespace ( line number 358)

Reply:
Done